optlang/cplex_interface.py

```diff
@@ -31,14 +31,17 @@
 
 log = logging.getLogger(__name__)
 
+from future.utils import raise_from
```

Comment:
`future` is built on top of six and I find it a lot nicer in many cases but sure, I can drop it.